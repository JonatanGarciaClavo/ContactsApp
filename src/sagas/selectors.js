export const contactSelector = state => state.contact;
export const contactCardSelector = state => state.contactCard;
export const contactListSelector = state => state.contactList;
