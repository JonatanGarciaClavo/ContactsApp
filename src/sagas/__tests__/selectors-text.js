import { contactSelector, contactCardSelector, contactListSelector } from '../selectors';

describe('Test selectors', () => {
  it('returns contact state', () => {
    expect(contactSelector({})).toMatchSnapshot();
  });
  it('returns contact card state', () => {
    expect(contactCardSelector({})).toMatchSnapshot();
  });
  it('returns contact list state', () => {
    expect(contactListSelector({})).toMatchSnapshot();
  });
});
