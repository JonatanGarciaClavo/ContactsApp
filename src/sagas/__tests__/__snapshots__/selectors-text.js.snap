exports[`Test selectors returns contact card state 1`] = `undefined`;

exports[`Test selectors returns contact list state 1`] = `undefined`;

exports[`Test selectors returns contact state 1`] = `undefined`;
